### DISCUSSION TITLE
Will Laravel 8 ever support Vite?

### DISCUSSION THREAD

**Discussion by amacsmith** [laravel/framework#43953] [Q&A]
Just wondering because at some point I would like to use Vite. However, just planning on whether to expect support in Laravel 8 or start planning a migration to Laravel 9.

**Comment by erikn69** [comment]
Laravel 8.x is reaching his EOL, don't expect support 
![image](https://user-images.githubusercontent.com/4933954/187941808-4f07205b-2a1c-445e-8b35-00e7ee611d56.png)
